networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/peer/DisconnectReason.java

```diff
@@ -23,6 +23,7 @@
   UNABLE_TO_VERIFY_NETWORK(GoodbyeMessage.REASON_UNABLE_TO_VERIFY_NETWORK, true),
   TOO_MANY_PEERS(GoodbyeMessage.REASON_TOO_MANY_PEERS, false),
   REMOTE_FAULT(GoodbyeMessage.REASON_FAULT_ERROR, false),
+  NO_PING_RESPONSES(GoodbyeMessage.REASON_FAULT_ERROR, false),
```

Comment:
This is outside the scope of this PR - but seems like this eth2-specific logic (`GoodbyeMessage` / PING behavior) should be in the `eth2` package.